### DISCUSSION TITLE
TPA GUI - Feature Request

### DISCUSSION THREAD

**Discussion by LucidusMC** [Zrips/CMI#1956] [Feature Requests]
It would be great if you added a TPA GUI to make teleporting faster. Thanks.

**Comment by Zrips** [comment]
How does that make it faster? I mean, tpa only requires you to enter players name which can be finished with Tab, so having a UI with bunch of player heads can take allot longer then simply entering desired name

**Comment by LucidusMC** [comment]
I would say it's about accessibility, my younger users struggle with names and order of users, plus it is just a nice way to see who is online. Thanks.
